### PR TITLE
more type checks

### PR DESCRIPTION
**Pull request by erwinmombay**

### CODE REVIEW

File: extensions/amp-install-serviceworker/0.1/amp-install-serviceworker.js [hunk 1]
```diff
@@ -30,6 +30,17 @@ const TAG = 'amp-install-serviceworker';
  * of the current page.
  */
 class AmpInstallServiceWorker extends AMP.BaseElement {
+
+  /** @param {!AmpElement} element */
+  constructor(element) {
+    super(element);
+
+    /** @private {boolean} */
+    this.insertedIframe_ = false;
```
Comment (erwinmombay) [inline extensions/amp-install-serviceworker/0.1/amp-install-serviceworker.js]: @cramforce this doesn't seem to be read anywhere. can we removed this?

File: extensions/amp-share-tracking/0.1/amp-share-tracking.js [hunk 2]
```diff
@@ -33,6 +33,18 @@ const SHARE_TRACKING_NUMBER_OF_BYTES = 6;
  * @visibleForTesting
  */
 export class AmpShareTracking extends AMP.BaseElement {
+
+  /** @param {!AmpElement} element */
```
Comment (erwinmombay) [inline extensions/amp-share-tracking/0.1/amp-share-tracking.js]: @muxin please review

File: extensions/amp-share-tracking/0.1/amp-share-tracking.js [hunk 3]
```diff
@@ -131,7 +141,7 @@ export class AmpShareTracking extends AMP.BaseElement {
    * Get a random bytes array that has 48 bits (6 bytes).
    * Use win.crypto.getRandomValues if it is available.
    * Otherwise, use Math.random as fallback.
-   * @return {!Promise<string>}
+   * @return {!Uint8Array}
```
Comment (muxin) [inline extensions/amp-share-tracking/0.1/amp-share-tracking.js]: Thanks for fixing this!

File: extensions/amp-share-tracking/0.1/amp-share-tracking.js (hunk 2 above)
Comment (muxin) [inline extensions/amp-share-tracking/0.1/amp-share-tracking.js]: LGTM for amp-share-tracking

File: extensions/amp-install-serviceworker/0.1/amp-install-serviceworker.js (hunk 1 above)
Comment (cramforce) [inline extensions/amp-install-serviceworker/0.1/amp-install-serviceworker.js]: Totally

**Comment by cramforce** [comment]
LGTM

**Pull request merged**